### PR TITLE
19271 - continuation in email

### PR DESCRIPTION
**Pull request by ketaki-deodhar**
*Issue #:* /bcgov/entity#19271

*Description of changes:*

- create email templates
- create email processor for continuation in
- add unit tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            '?type=certificateOfContinuation',
+            headers=headers
+        )
+        if certificate.status_code != HTTPStatus.OK:
+            logger.error('Failed to get certificate of continuation for filing: %s', filing.id)
+        else:
+            certificate_encoded = base64.b64encode(certificate.content)
+            pdfs.append(
+                {
+                    'fileName': 'Certificate of Continuation.pdf',
```
Comment (vysakh-menon-aot) [inline queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py:110]: Can you confirm the name is `Certificate of Continuation` in UI design its just `Certificate`
image

Comment (ketaki-deodhar) [reply queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py:110]: Referred to [Email spreadsheet](https://docs.google.com/spreadsheets/d/17wMY9znmWMr2RoXH9S2elffC8uE7iifUGB5Hu3VVFpE/edit?gid=0#gid=0) 
I will check.

Comment (ketaki-deodhar) [reply queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py:110]: Andy confirmed it should be 'Certificate of Continuation'. He also updated the designs.

Comment (vysakh-menon-aot) [reply queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py:110]: @ketaki-deodhar can you update here https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/core/meta/filing.py#L273 from `certificate` to `certificateOfContinuation`

File: queue_services/entity-emailer/src/entity_emailer/email_templates/CONT-IN-COMPLETED.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  <body>
+    <table class="body-table" role="presentation">
+      <tr>
+        <td>
+          [[header.html]]
+
+          <div class="container">
+            <p class="title-message bold">
+              <span>Continuation Documents from the Business Registry<span>
```
Comment (vysakh-menon-aot) [inline queue_services/entity-emailer/src/entity_emailer/email_templates/CONT-IN-COMPLETED.html:21]: As per spreadsheet this should be `You have successfully registered your business with BC Business Registry.`

**Review by vysakh-menon-aot — APPROVED** [review]

**Pull request merged**